stable/luigi/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: luigi
+version: 0.1.0
```

Comment:
Can you add at least one maintainer. The name needs to be a GitHub username so we can contact the person on issues and PRs that come in about the chart.